numpy/core/tests/test_ufunc.py

```diff
@@ -43,7 +44,10 @@ def test_extobj_refcount(self):
 
 class TestUfunc(object):
     def test_pickle(self):
-        import pickle
+        if sys.version_info[:2] in ((3, 7), (3, 6)):
+            import pickle5 as pickle
+        else:
+            import pickle
```

Comment:
I don't think the local import is necessary. I think this can be moved to the top level as:

```
from numpy.core.numeric import pickle
```